projects/bp-strapi/config/plugins.js

```diff
@@ -27,4 +27,10 @@ module.exports = ({ env }) => ({
       jwtSecret: env("JWT_SECRET"),
     },
   },
+  sentry: {
+    enabled: !!env('SENTRY_DSN'),
```

Comment:
```suggestion
    enabled: env.bool('SENTRY_DSN'),
```

Does this work? Strapi Docu for this: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.html#configuration-using-environment-variables
Probably not, but lets try it mabye :D

Reply:
No, because I want to test if it exists, not if the boolean value is true. If the env var doesn't exists this should be undefined and I use !! to assert that it exists. It works like a `Boolean(...)`

Reply:
Alrighty!